Integrations/FeedRecordedFuture/FeedRecordedFuture_test.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from collections import OrderedDict
+from FeedRecordedFuture import get_indicator_type, split_hash_context, get_indicator_context
+
```

Comment:
If indicator types is reduced to just `File`, then you will need to adjust the unit tests accordingly as well.